man/journald.conf.xml

```diff
@@ -139,6 +139,50 @@
         <literal>us</literal>. To turn off any kind of rate limiting,
         set either value to 0.</para>
 
+        <para>The <varname>RateLimitBurst=</varname> is further modified
+        by the log() of the available disk space for the journal.</para>
+
+        <table>
+          <title>Example <varname>RateLimitBurst=</varname> rate
+            modifications by the available disk space</title>
+          <tgroup cols='2'>
+            <colspec colname='freespace' />
+            <colspec colname='multiplier' />
+            <thead>
+              <row>
+                <entry>Available Disk Space</entry>
+                <entry>Burst Multiplier</entry>
+              </row>
+            </thead>
+            <tbody>
+              <row>
+                <entry>&lt;= 1MB</entry>
+                <entry>1</entry>
+              </row>
+              <row>
+                <entry>&lt;= 16MB</entry>
+                <entry>2</entry>
+              </row>
+              <row>
+                <entry>&lt;= 256MB</entry>
+                <entry>3</entry>
+              </row>
+              <row>
+                <entry>&lt;= 4GB</entry>
+                <entry>4</entry>
+              </row>
+              <row>
+                <entry>&lt;= 64GB</entry>
+                <entry>5</entry>
+              </row>
+              <row>
+                <entry>&lt;= 1TB</entry>
+                <entry>6</entry>
+              </row>
```

Comment:
and above 1TB ?

Reply:
I have to include the actual calculation or provide a complete table to infinity. If an end-user has more than 1TB of disk space, I think they should be able to figure out it will be at least 6 and they need to find some other way to filter messages.